raiden/api/python.py

```diff
@@ -576,6 +576,63 @@ def _set_total_deposit_preconditions(
             )
             raise InsufficientFunds(msg)
 
+    def settle_light(
+        self,
+        registry_address: PaymentNetworkID,
+        token_address: TokenAddress,
+        creator_address: Address,
+        partner_address: Address,
+        channel_identifier: typing.ChannelID,
+        signed_settle_tx: typing.SignedTransaction = None,
+    ):
+        """Settle a channel opened with `partner_address` for the given
+        `token_address`.
+        Race condition, this can fail if channel was settled externally.
+        """
+
+        chain_state = views.state_from_raiden(self.raiden)
+
+        channels_to_settle = ChannelValidator.validate_and_get_channels_to_settle(
+            token_address=token_address,
+            creator_address=creator_address,
+            partner_address=partner_address,
+            registry_address=registry_address,
+            chain_state=chain_state)
+
+        # get the channel to settle
+        channel_iterator = filter(lambda channel:
+                                  channel.our_state.address == creator_address
+                                  and channel.partner_state.address == partner_address
+                                  and channel.token_address == token_address
+                                  and channel.identifier == channel_identifier, channels_to_settle)
+
+        channel_list = list(channel_iterator)
+
+        if len(channel_list) <= 0:
```

Comment:
You can change this to `if channel_list`

Reply:
Good to know

Reply:
Should be ```if not channel_list``` right?

Reply:
Yeah sorry my bad

Reply:
Done